src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -55793,7 +55793,7 @@ static Action M(Action a)
             comp.VerifyDiagnostics();
         }
 
-        [Fact]
+        [ConditionalFact(typeof(IsRelease))]
```

Comment:
What motivated the change?

Reply:
> What motivated the change?

There is additional validation of lambdas (and method groups) in `Binder.CreateAnonymousFunctionConversion()` (and `Binder.CreateMethodGroupConversion()`) that made these tests unnecessarily expensive in DEBUG builds.

Reply:
Consider leaving a comment explaining why we skip (it's just slow)

---
In reply to: [612782049](https://github.com/dotnet/roslyn/pull/52448#discussion_r612782049) [](ancestors = 612782049)